packages/block-editor/src/store/selectors.js

```diff
@@ -1963,7 +1963,14 @@ const buildBlockTypeItem =
 export const getInserterItems = createSelector(
 	( state, rootClientId = null ) => {
 		const buildReusableBlockInserterItem = ( reusableBlock ) => {
-			const icon = symbol;
+			const icon =
+				reusableBlock.wp_pattern_sync_status === 'synced' ||
```

Comment:
I don't think we assign a sync status of `synced` anywhere currently, `synced` is always undefined, so I wonder if we need this extra check?

Reply:
Oh I was just following the logic in [line 1990](https://github.com/WordPress/gutenberg/pull/53327/files#diff-aa92c48095c35f17feb834dd919baf52274a73c7b1180ff5acce7b6087b126cfR1990). But yeah, we can simplify this.

Reply:
I don't see that logic, did you paste the correct link, it just circles back to this diff?

Reply:
I simply mean that we define `syncStatus` by referencing `wp_pattern_sync_status` in `syncStatus: reusableBlock.wp_pattern_sync_status` rather than hardcoding it. (Which implicitly means that it depends on the `wp_pattern_sync_status` variable)

Reply:
I don't see that logic on 1990?  but probably worth simplifying this to avoid any confusion.

Reply:
At first glance, this looked like something left behind from when we were assigning a value for synced status on patterns. I agree, this check doesn't appear to be required anymore.

Reply:
I just gave it a try in my environment but seems like unsynced patterns are also returned by the `getReusableBlocks` call, so the check is still needed. Is it because I'm on an older core? Can any of you reproduce this? 🙇‍♂️

Reply:
Sorry for the confusion, I was meaning the first condition in the check, only the part highlighted by Glen's original comment. It works as expected for me removing just that conditional.

```diff
diff --git a/packages/block-editor/src/store/selectors.js b/packages/block-editor/src/store/selectors.js
index f5002f2b027..45955366f1b 100644
--- a/packages/block-editor/src/store/selectors.js
+++ b/packages/block-editor/src/store/selectors.js
@@ -1963,14 +1963,12 @@ const buildBlockTypeItem =
 export const getInserterItems = createSelector(
 	( state, rootClientId = null ) => {
 		const buildReusableBlockInserterItem = ( reusableBlock ) => {
-			const icon =
-				reusableBlock.wp_pattern_sync_status === 'synced' ||
-				! reusableBlock.wp_pattern_sync_status
-					? {
-							src: symbol,
-							foreground: 'var(--wp-block-synced-color)',
-					  }
-					: symbol;
+			const icon = ! reusableBlock.wp_pattern_sync_status
+				? {
+						src: symbol,
+						foreground: 'var(--wp-block-synced-color)',
+				  }
+				: symbol;
 			const id = `core/block/${ reusableBlock.id }`;
 			const { time, count = 0 } = getInsertUsage( state, id ) || {};
 			const frecency = calculateFrecency( time, count );

```

Reply:
Oh oops! I completely missed that we don't use the `synced` enum anymore 🤦 .

Reply:
All good. Small price to pay given all the chopping and changing throughout our pattern explorations 👍